encord/objects/ontology_labels_impl.py

```diff
@@ -105,6 +107,7 @@ def __init__(
         self._frame_metadata: defaultdict[int, Optional[DICOMSliceMetadata]] = defaultdict(lambda: None)
 
         self._classifications_to_frames: defaultdict[Classification, Set[int]] = defaultdict(set)
+        self._classifications_to_ranges: defaultdict[Classification, RangeManager] = defaultdict(RangeManager)
```

Comment:
This will be the equivalent of `self._classifications_to_frames`, but will be used for audio (and probably text and pdf)

Reply:
FYI, this is used to keep track of the range of "frames" that a classification applies to. This is important when we want to check whether there are overlapping ranges.